class/UserAccessManager.php

```diff
@@ -155,6 +155,9 @@ public function getFromCache($sKey)
     public function getUser($sId)
     {
         if (!isset($this->_aUsers[$sId])) {
+            if (!function_exists('get_userdata')) {
```

Comment:
This shouldn't part of the UserAccessManager class and should move to the user-access-manager.php file which loads the class.